packages/gatsby/index.d.ts

```diff
@@ -293,11 +293,33 @@ export interface GatsbyNode {
    *
    * See also the documentation for `createNode`
    * and [`createNodeField`](https://www.gatsbyjs.org/docs/actions/#createNodeField)
+   * @param {object} $0
+   * @param {object} $0.node A node object.
+   * @param {object} $0.actions
+   * @param {function} $0.actions.createNode Create a new node.
+   * @param {function} $0.actions.createNodeField Extend another node. The new node field is placed under the fields key on the extended node object.
    * @example
-   * exports.onCreateNode = ({ node, actions }) => {
-   *   const { createNode, createNodeField } = actions
-   *   // Transform the new node here and create a new node or
-   *   // create a new node field.
+   * exports.onCreateNode = ({ node, getNode, actions }) => {
+   *   const { createNodeField } = actions;
+   *
+   *   if (node.internal.type === `Mdx`) {
+   *     const nodePath = node.fileAbsolutePath;
+   *
+   *     if (nodePath.match(/\/blog\//)) {
+   *       const postSlug = createFilePath({
+   *         node,
+   *         getNode,
+   *         basePath: `src/content`,
+   *         trailingSlash: true,
+   *       });
+   *
+   *       createNodeField({
+   *         node,
+   *         name: `slug`,
+   *         value: polishSlug(`/blog/${postSlug}/`),
```

Comment:
Where is `polishSlug` coming from?

Reply:
it is function from my project, I forgot to remove it for the example, will do so now